### PR TITLE
feat: add share button on artist page

### PR DESCRIPTION
**Pull request by dimatretyak**
The type of this PR is: **Chore**
Notion card: [Link](https://www.notion.so/artsy/Add-share-button-to-artist-page-in-app-d38f0525f2d7442c8e948ad584390d78)

### iOS
https://user-images.githubusercontent.com/3513494/150170351-5906bbb6-272a-4546-a037-a3d490e809a6.mp4

#### Android
https://user-images.githubusercontent.com/3513494/150170457-ed240553-6a1c-46b4-9e38-0bc5c3dc9ba7.mp4

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [ ] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Add share button on artist page - dimatretyak

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (Add share button on artist page - dimatretyak)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 64f8c96d90cc9ab5bfb06b93fce91cd6d9fbea8c
</p>

**Review by gkartalis — DISMISSED** [review]
🔥

File: src/lib/Components/ShareSheet/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+export const getShareMessage = (artistNames: string[], title?: string) => {
+  const names = artistNames.slice(0, 3).join(", ")
+
+  if (title) {
+    return `${title} by ${names} on Artsy`
+  }
+
+  return `${names} on Artsy`
```
Comment (MounirDhahri) [inline src/lib/Components/ShareSheet/helpers.ts:15]: **praise:** very thoughtful

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Thanks! Please feel free to merge

**Review by MounirDhahri — APPROVED** [review]

**Comment by dimatretyak** [comment]
@MounirDhahri @gkartalis can you please review the changes again? Now the button is floating + clarified moments about analytics

**Review by MounirDhahri — DISMISSED** [review]
Thanks!

**Review by gkartalis — APPROVED** [review]

**Pull request merged**